api/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.17.3",
+    "jest": "^27.5.1",
+    "pg": "^8.7.3"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "dev": "nodemon index.js"
```

Comment:
So this is correct, but your nodemon does not run because you named it under the scripts "dev" and you use the command "npm start" in your yaml file, therefore it only runs node and not nodemon.